src/libraries/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Intrinsics.cs

```diff
@@ -0,0 +1,429 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
+
+namespace System.Text.Json
+{
+    internal static partial class JsonReaderHelper
+    {
+        // https://tools.ietf.org/html/rfc8259
+        // Does the span contain '"', '\',  or any control characters (i.e. 0 to 31)
+        // IndexOfAny(34, 92, < 32)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static int IndexOfQuoteOrAnyControlOrBackSlash(this ReadOnlySpan<byte> span)
+        {
+            if (Sse2.IsSupported || AdvSimd.Arm64.IsSupported || Vector.IsHardwareAccelerated)
+            {
+                return IndexOfOrLessThan(
+                        ref MemoryMarshal.GetReference(span),
+                        JsonConstants.Quote,
+                        JsonConstants.BackSlash,
+                        lessThan: JsonConstants.Space,
+                        span.Length);
+            }
+            else
+            {
+                return IndexOfOrLessThanNonVector(span);
+            }
+        }
+
+        // Borrowed and modified from SpanHelpers.Byte:
+        // https://github.com/dotnet/runtime/blob/a196d534495a1654ffd158211ca8761b1eba8c05/src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs#L916-L1207
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
We should not need to be adding `AggressiveOptimization` to method like these with crossgen2.

Reply:
Removed.

I assume crossgen2 is the default now and everything in package uses it?